.eslintignore

```diff
@@ -5,3 +5,6 @@
 
 # The parsers are auto-generated
 /packages/yarnpkg-parsers/sources/grammars/*.js
+
+/packages/yarnpkg-core/sources/ZipConvertWorkerSource.js
```

Comment:
It would require to run a build after cloning the repository, right? Perhaps it should be checked-in (just like we do with the PnP hook).

Reply:
It is checked in, this is to get eslint to leave it alone

Reply:
Mistook it for a gitignore 🤦‍♀️